R/utils-MakeTargetData.R

```diff
@@ -0,0 +1,31 @@
+#' Helper function to compile "long" group metadata
+#'
+#' @description
+#'
+#' Used to create percentage values on enrollment data, such as site activation and
+#' and participantment enrollment, and pasting together an appropriate n/N (xx.x%) value
+#' as well.
+#'
+#' @param data The input dataframe
+#' @param current_col Column that represents site count or participant count
+#' @param target_col Column that represents the target count for site/participants
+#' @param perc_val Name of column that will contain the numeric percentage value on enrollment
+#' @param perc_str_val Name of column that will contain the n/N (xx.x%) string
+#'
+#' @return
+#' @export
+#'
+#' @examples
+MakeTargetData <- function(data, current_col, target_col, perc_val, perc_str_val) {
```

Comment:
A utility function has been added to do the perc calculations and generate the perc string (to be run twice in `STUDY.yaml` for both participants and sites

Reply:
Update parameter naming convention to `typeFunctionName`.